src/data/stig_multipliers.ts

```diff
@@ -386,6 +386,70 @@ const SUPPORT_STIG_MULTIPLIERS = Object.freeze({
 });
 
 const DPS_STIG_MULTIPLIERS = Object.freeze({
+  "Allen Poe": {
+    T: [
+      new Multiplier({
+        active: true,
+        name: "Allen Poe T",
+        type: Type.CritRate,
+        value: 18,
+      }),
+      new Multiplier({
+        active: false,
+        name: "Allen Poe T: if 2 or more enemies, convert 2%*n of the bonus crit rate to 4%*n phys DMG (max 5)",
+        type: Type.PhysDealt,
+        value: 20,
+      }),
+    ],
+    M: [
+      new Multiplier({
+        active: true,
+        name: "Allen Poe M: After spending no less than 3/33/63 sp, boost crit DMG for basic ATKs (charged/combo included)",
```

Comment:
Maybe make the description for each the respective SP values? Ex. have this one say just "3 sp", the 60% one should say "33 sp", etc.

Fwiw I seem to remember from watching a few SN rotations that it's quite typical to hit the 33 SP threshold even on the first ult in a run as long as you charge first

Reply:
Incorporated